packages/react-native/React/Base/RCTConvert.h

```diff
@@ -154,7 +154,7 @@ typedef BOOL css_backface_visibility_t;
 + (RCTTextDecorationLineType)RCTTextDecorationLineType:(id)json;
 
 #if TARGET_OS_OSX // [macOS
-+ (NSString *)accessibilityRoleFromTraits:(id)json;
++ (NSString *)accessibilityRoleFromTraits:(id)json usingAriaMappings:(BOOL) useAriaMappings;
```

Comment:
Sorry, I'm gonna be dumb. I think the style is `useAriaMappings:(BOOL)useAriaMappings`. Redundant, yes, hence why Swift makes it nicer. Sorry for the double change :/